apps/address-service/domains/common/utils/services/suggest/SuggestionKeystoneApp.js

```diff
@@ -56,15 +56,19 @@ class SuggestionKeystoneApp {
             }
 
             // 1. Detect the suggestion provider
-            const suggestionProvider = providerDetector.getProvider(geo)
+            const suggestionProvider = getSuggestionsProvider(geo)
 
```

Comment:
Could you please also move JSDoc comments for local variables to header of the function and describe an expected shape of `req` variable. It's easier to understand what this method expects without digging into it's implementation.

Reply:
Done